blockchain/util/crypto.py

```diff
@@ -204,6 +204,28 @@ def sign_verification_record(signatory,
     return block_info
 
 
+def sign_subscription(signatory, subscription, private_key_string, public_key_string):
```

Comment:
Please add complete documentation (include what elements are being signed) - see other functions in file.